app/services/katello/event_daemon/monitor.rb

```diff
@@ -8,6 +8,7 @@ def initialize(service_classes)
       def start
         error = nil
         status = nil
+        check_services(error, status)
```

Comment:
added this because I was getting a FAIL on the first request

Reply:
Was that problematic in some way?

Reply:
Isn't it always problematic to get a FAIL on a hammer ping?  (For our friend the original bug reporter it seems to be.)

Reply:
You're saying hammer ping always fails on the first request without this addition?

Reply:
Well, I'm doing `curl localhost:3000/api/v2/ping` but yes.  It fails for the first few seconds.

Reply:
I think it's because `check_services` is the only place that does `service_class.run`.  Seems like that should happen when the daemon is started..